x-pack/plugins/apm/public/components/shared/charts/Histogram/index.js

```diff
@@ -235,6 +237,17 @@ export class HistogramInner extends PureComponent {
               y={() => 1}
             />
           </XYPlot>
+
+          {legends && (
+            <Legends
```

Comment:
Would like to disable the click to hide option because there's only a single line

![Kapture 2020-06-04 at 21 07 19](https://user-images.githubusercontent.com/4104278/83800368-a23edf00-a6a7-11ea-9f49-f63dbffc8a88.gif)